lambdas/build-cri-oauth-request/src/main/java/uk/gov/di/ipv/core/buildcrioauthrequest/BuildCriOauthRequestHandler.java

```diff
@@ -323,11 +328,12 @@ private JWEObject signEncryptJar(
         return AuthorizationRequestHelper.createJweObject(rsaEncrypter, signedJWT);
     }
 
-    private EvidenceRequest getEvidenceRequestForF2F(List<VerifiableCredential> vcs) {
+    private EvidenceRequest getEvidenceRequestForF2F(
+            List<VerifiableCredential> vcs, Vot minimumRequestedVotsByStrength) {
         var gpg45Scores = gpg45ProfileEvaluator.buildScore(vcs);
         List<Gpg45Scores> requiredEvidences =
                 gpg45Scores.calculateGpg45ScoresRequiredToMeetAProfile(
```

Comment:
Could you add a couple of test cases for this in `Gpg45ScoresTest.java` to cover the L1A profile?